src/Test.FunctionalTests.BinSkim.Driver/BaselineTests.cs

```diff
@@ -438,7 +438,7 @@ private SarifLog RunRules(StringBuilder sb, string inputFileName, Sarif.SarifVer
                 ConfigurationFilePath = "default",
                 SarifOutputVersion = Sarif.SarifVersion.Current,
                 TargetFileSpecifiers = new string[] { inputFileName },
-                Traces = Array.Empty<string>(),
+                Trace = Array.Empty<string>(),
```

Comment:
>Trace [](http://example.com/codeflow?start=16&length=5)

Why is this required? Can you remove it?

Reply:
removed.